components/inputs/input-text.js

```diff
@@ -533,14 +543,16 @@ class InputText extends FormElementMixin(SkeletonMixin(RtlMixin(LitElement))) {
 		this._hovered = false;
 	}
 
-	_handleSlotChange(e) {
-		const container = e.target.parentNode;
-
-		// requestUpdate needed for legacy Edge
-		this.requestUpdate();
-		this.updateComplete.then(() => {
-			this._updateInputLayout(container);
-		});
+	_handleSlotChange() {
+		// requestUpdate needed for legacy-Edge
+		if (navigator.userAgent.indexOf('Edge/') > -1) {
+			this.requestUpdate();
```

Comment:
This was causing a double-render in all browsers, so putting in a temporary user agent check here avoids it. We can delete this soon.